src/app/qgsmeasuredialog.cpp

```diff
@@ -364,10 +376,14 @@ void QgsMeasureDialog::updateUi()
 
   if ( mMeasureArea )
   {
-    if ( !mCanvas->mapSettings().destinationCrs().isValid() )
+    if ( mCartesian->isChecked() || !mCanvas->mapSettings().destinationCrs().isValid() )
     {
-      // no CRS => no units, newb!
-      toolTip += "<br> * " + tr( "No map projection set, so area is calculated using Cartesian calculations." );
+      toolTip += "<br> * ";
+      if ( mCartesian->isChecked() )
+        toolTip += tr( "Wanted Cartesian" );
+      else
+        toolTip += tr( "No map projection set" );
+      toolTip += tr( ", so area is calculated using Cartesian calculations." );
```

Comment:
I think you'll need to combine these strings including the duplicate portion, or it won't necessarily work correctly with translations.